cluster/saltbase/pillar/privilege.sls

```diff
@@ -1,2 +1,2 @@
 # If true, allow privileged containers to be created by API
-allow_privileged: false
+allow_privileged: true
```

Comment:
Why leave this in the salt? You removed it from coreos/systemd and ansible. It seems like we should remove it everywhere or just set it to true everywhere.